multiplex_immuno_processing/generate_contact_sheet_gif.py

```diff
@@ -0,0 +1,72 @@
+import os
+import numpy as np
+import glob
+import re
+import tifffile
+from PIL import Image, ImageDraw
```

Comment:
aicsimageio has a timeseries writer that can write gifs. I'd recommend using that.

Reply:
I didn't include a change for this yet, but we can do this in a separate branch